src/installer.ts

```diff
@@ -114,14 +114,18 @@ async function queryLatestMatch(versionSpec: string) {
 }
 
 async function getAvailableVersions() {
-  const rest = new restm.RestClient("setup-deno");
-  const data =
-    (
-      await rest.get<{ name: string }[]>(
-        "https://denolib.github.io/setup-deno/release.json"
-      )
-    ).result || [];
-  return data.map(v => v.name);
+  // a temporary workaround until a Release API is provided. (#11)
+  const httpc = new HttpClient("setup-deno");
+  const body = await (
+    await httpc.get(
+      "https://raw.githubusercontent.com/denoland/deno/master/Releases.md"
+    )
+  ).readBody();
+  const matches = body.matchAll(/### (v\d+\.\d+\.\d+)/g);
+  let releases = [...matches].map(m => m[1]);
+
+  // exclude v0.0.0

```

Comment:
Can we specify to exclude the `v0.0.0` version?

```ts
releases.filter(v => v !== "v0.0.0")
```

Worried about changes in Release.md in the future